jd-geek-source/src/app/app.module.ts

```diff
@@ -41,6 +47,12 @@ import { ShoppingCartComponent } from "./components/shopping-cart/shopping-cart.
     SearchIconComponent,
     ShoppingCartIconComponent,
     ShoppingCartComponent,
+    AngleLeftIconComponent,
+    AngleRightIconComponent,
+    FilledStarIconComponent,
+    EmptyStarIconComponent,
+    HalfStarIconComponent,
+    CartPlusIconComponent,
```

Comment:
if you are going to be creating a bunch of these, it might be easier to create a feature module to house all these so that the app module doesn't get swamped.